hash/dc.xml

```diff
@@ -23459,10 +24092,10 @@ cfr. https://github.com/mamedev/mame/issues/9106 for more info.
 		<description>GD Drive Repair Program v0.1 (World)</description>
 		<year>1998</year>
 		<publisher>Sega</publisher>
-		<notes><![CDATA[
+		<notes>
 Check-GD program
 Stuck with a Sega copyright printed on screen
-]]></notes>
+</notes>
```

Comment:
This won't compile.

Reply:
These were stripped by the xml parser, noted and will restore.